tool/tsh/tsh.go

```diff
@@ -1895,11 +1905,22 @@ func onListClusters(cf *CLIConf) error {
 	return nil
 }
 
+func labelsString(labels map[string]string) string {
```

Comment:
We have [sortedLabels](https://github.com/gravitational/teleport/blob/v9.1.3/tool/tsh/tsh.go#L1568) which we already use to display labels in other places, e.g. take a look at [showAppsAsText](https://github.com/gravitational/teleport/blob/v9.1.3/tool/tsh/tsh.go#L1621). Can you use the same approach instead of making a new function?

Reply:
Ah didn't see that function, will do